BossMod/Components/Knockback.cs

```diff
@@ -137,6 +139,8 @@ protected IEnumerable<(WPos from, WPos to)> CalculateMovements(BossModule module
                     Kind.AwayFromOrigin => from != s.Origin ? (from - s.Origin).Normalized() : new(),
                     Kind.DirForward => s.Direction.ToDirection(),
                     Kind.DirLeft => s.Direction.ToDirection().OrthoL(),
+                    Kind.DirRight => s.Direction.ToDirection().OrthoR(),
+                    Kind.TowardsOrigin => s.Direction.ToDirection().OrthoL().OrthoL(),
```

Comment:
Should be simply `-s.Direction.ToDirection()`
Also - minor thing - please keep enum and switch order the same.

Reply:
I guess I can change that